db/controllers/articles.controller.js

```diff
@@ -32,11 +33,22 @@ exports.getArticleComments = (req, res, next) => {
 
 exports.getArticleById = (req, res, next) => {
   const { article_id } = req.params;
-  return selectArticleById(article_id)
-    .then((article) => {
-      res.status(200).send({ article });
-    })
-    .catch(next);
+  const { query } = req.query;
+  if (query === "comment_count") {
```

Comment:
This behaviour where an article is retrieved by its id also has the comment count property shouldn't rely being given a query of `comment_count`
When a user makes a request to `/api/articles/article_id` the response should now have the additional property of comment count

Reply:
Thanks, sorted.